### PR TITLE
add onclite and crownlte, remove a few extra spaces and change status of sdcard for 3 devices (tested, works) and a few strings updated

### PR DESCRIPTION
**Pull request by FakeShell**

### CODE REVIEW

File: content/devices/angelica.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Please download the belowed needed files and tools:
     - The first boot may take longer, and at least one spontaneous reboot is expected during the process
     - You should be greeted with the lock screen, the default password is `1234`
 
-Congratulations, if everything went well, now you should be running Droidian.
```
Comment (sanathusk) [inline content/devices/angelica.md:79]: The files inside content/devices are now generated by the generate_device_pages.js script, if the changes are present in template files, the changes shoul relect here.

File: content/devices/crownlte.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+---
+draft: false
```
Comment (sanathusk) [inline content/devices/crownlte.md:2]: Please add a template file instead, Please refer readme file for updated instructions.

File: content/devices/beryllium.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ Please download the belowed needed files and tools:
     sudo systemctl enable enable-ipa sudo systemctl reboot
     ```
 
-Congratulations, if everything went well, now you should be running Droidian.
+Congratulations, if everything went well you should be booting into Droidian.
```
Comment (sanathusk) [inline content/devices/beryllium.md:76]: The files inside content/devices are now generated by the generate_device_pages.js script, if the changes are present in template files, the changes shoul relect here.

File: content/devices/onclite.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
+---
+draft: false
+title: Xiaomi Redmi 7 (onclite)
+---
+> **Make a backup now, as your device will be wiped.**
+## Before you proceed
```
Comment (sanathusk) [inline content/devices/onclite.md:6]: Please add a template file instead, Please refer readme file for updated instructions.

File: content/devices/perseus.md [hunk 5]
```diff
@@ -59,7 +59,7 @@ Flash recovery (using Computer)
     - The first boot may take longer, and at least one spontaneous reboot is expected during the process
     - You should be greeted with the lock screen, the default password is `1234`
 
-Congratulations, if everything went well, now you should be running Droidian.
+Congratulations, if everything went well you should be booting into Droidian.
```
Comment (sanathusk) [inline content/devices/perseus.md:62]: The files inside content/devices are now generated by the generate_device_pages.js script, if the changes are present in template files, the changes should reflect here.

File: data/supported-devices/devices/xiaomi_onclite.yml [hunk 6]
```diff
@@ -0,0 +1,181 @@
+manufacturer: Xiaomi
```
Comment (sanathusk) [inline data/supported-devices/devices/xiaomi_onclite.yml:1]: Please add a template file instead, Please refer readme file for updated instructions.

File: templates/star2lte_template.md [hunk 7]
```diff
@@ -66,7 +66,7 @@ Please download the belowed needed files and tools:
     - The first boot may take longer, and at least one spontaneous reboot is expected during the process
     - You should be greeted with the lock screen, the default password is `1234`
 
-Congratulations, if everything went well, now you should be running Droidian.
```
Comment (sanathusk) [inline templates/star2lte_template.md:69]: These changes should be sufficient , the changes will reflect in files generated inside content/devices folder.

File: content/devices/angelica.md (hunk 1 above)
Comment (FakeShell) [reply content/devices/angelica.md:79]: i changed this piece of text in templates/* then ran generate_device_pages.js

File: content/devices/beryllium.md (hunk 3 above)
Comment (FakeShell) [reply content/devices/beryllium.md:76]: i changed this piece of text in templates/* then ran generate_device_pages.js

**Comment by FakeShell** [comment]
should be fine now

**Review by sanathusk — APPROVED** [review]

**Pull request merged**